cmd/crud/handlers/health.go

```diff
@@ -11,11 +12,12 @@ import (
 
 // Health provides support for orchestration health checks.
 type Health struct {
+	Log      *log.Logger
```

Comment:
This struct field is unused since the logger is passed in to each handler. We should either pass it in to the handlers or have it as a struct field like this but not both.

Reply:
Excellent Catch!!

Reply:
Might not be applicable to this PR but: Why pass DB handle via struct and logger via arg?

Reply:
Ahh, I see b/c all handlers need a logger? And only some need DB handle?